src/components/ImportExport.js

```diff
@@ -22,6 +28,8 @@ class ImportExport extends React.Component {
     this.state = {
       selectedFiles: null,
       exportType: null,
+      dialogueMain: false,
```

Comment:
Let's name this `dialogOpen` or something a little more descriptive of it's purpose. `__Open` or `___IsOpen` is usually good for popups and dialogs. 

Also: dialogue????? Didn't realize you were so British.